resources/guis.yml

```diff
@@ -43,7 +43,7 @@ guis:
   - Windows
   - Mac
   - Linux
-  price: Free for non-commercial use
+  price: $49 / user / Free for non-commercial use and commercial startups less than a year old
```

Comment:
remove 'and commercial startups less than a year old' please :)

Reply:
Fixed. I also changed the "$49 / user" to be "$49/user" like SmartGit and Tower